kyuubi-hive-jdbc/src/main/java/org/apache/kyuubi/jdbc/hive/KyuubiConnection.java

```diff
@@ -937,6 +943,17 @@ public void close() throws SQLException {
     }
   }
 
+  public void closeOnLaunchEngineFailure(Thread logThread) throws SQLException {
```

Comment:
since `engineLogThread` is a global variable, why we need pass it here.

Reply:
thanks